erpnext/stock/doctype/item/item.json

```
@@ -4,7 +4,7 @@
  "allow_import": 1,
  "allow_rename": 1,
  "autoname": "field:item_code",
- "creation": "2013-05-03 10:45:46",
+ "creation": "2022-01-11 15:03:47.588455",
```

```suggestion
 "creation": "2013-05-03 10:45:46",
```
oof

--

this is fixed via https://github.com/frappe/frappe/pull/15763